src/RequestHandler.php

```diff
@@ -137,6 +137,13 @@ public function getNextSlave()
      */
     public function slaveAvailable(Slave $slave)
     {
+        if ($slave->isExpired()) {
+            $slave->close();
+            $this->output->writeln(sprintf('Restart worker #%d because it reached its TTL', $slave->getPort()));
```

Comment:
We should print that only in verbose mode, shouldn't we?

Reply:
I copied/paste the `max-requests` snippet, I have to say ^^
https://github.com/bpolaszek/php-pm/blob/513b845f9190cf70f97465da12b9fb7e7a01b970/src/RequestHandler.php#L230